frontend/package-lock.json

```diff
@@ -1,6 +1,10957 @@
 {
+  "name": "frontend",
+  "lockfileVersion": 2,
```

Comment:
Add some point your node and/or npm got updated to either v15.x.x and/or v7.x.x. I think we should maintain node and npm versions so that our package-lock.json file does not get updated every time some one on a different npm version runs `npm install`

Reply:
ok makes sense, but how do i fix that?

Reply:
You said you downloaded node manually right? In that case just run `npm install -g npm@6` to install major version 6.x.x

Reply:
And then run `npm install --package-lock-only` to convert the package-lock.json from lockfileVersion 2 to 1